packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -76,6 +76,7 @@ export class SearchBoxBase extends React.Component<ISearchBoxProps, ISearchBoxSt
       clearButtonProps,
       disableAnimation,
       iconProps,
+      role,
```

Comment:
Can you explicitly add this in ISearchBoxProps too?

Reply:
should be fixed now.